src/invariant/test/ConservationOfLumensTests.cpp

```diff
@@ -80,11 +80,26 @@ updateBalances(std::vector<LedgerEntry> entries, Application& app,
     REQUIRE(initialCoins + netChange == finalCoins);
 
     LedgerTxn ltx(app.getLedgerTxnRoot());
-    ltx.loadHeader().current().totalCoins += netChange;
+    auto& current = ltx.loadHeader().current();
+    if ((current.totalCoins < 0 && netChange < 0) ||
+        (current.totalCoins > 0 && netChange > 0))
+    {
+        CAPTURE(current.totalCoins);
```

Comment:
I did, in case there were other tests failing in the same way -- `CAPTURE()` only reports those values if the test fails.  However, I'm not attached to them; I can delete them while doing the other simplification deriving from `totalCoins` being non-negative.

Reply:
I've removed them along with the other simplifications.